index.html

```diff
@@ -5,11 +5,55 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>myExtension</title>
+    <link rel="stylesheet" href="./style.css">
 </head>
 
 <body>
+    <main>
+        <p class="name_content"></p>
+        <div class="weather_wrapper">
+            <div class="container">
+                <p class="desc"></p>
+                <img src="" class="weather_image" alt="weather image" />
+            </div>
+            <div class="container">
+                <div>
+                    <p class="temp"></p>
+                    <span class="degree_symbol">&#176;</span>
```

Comment:
You must have a fallback when using unicode because in some browser it is not recognized

Reply:
thanks, i'll keep this in mind and update accordingly.